cachemgr/test/boss.config

```diff
@@ -0,0 +1,50 @@
+# For CI testing on CircleCI, this file copied into /etc/boss on their Docker
+# container.
+
+[system]    
+fqdn = endpoint.test.boss    
+type = endpoint    
+    
+[consul]    
+cluster =    
+    
+[vault]    
+url = http://vault:8200    
+token = b4ee33a8-3336-ed37-3d9d-f7d5e87e2525    
```

Comment:
Is this a real token?

Reply:
It can't be can it. We don't have a vault on CI

Reply:
Not sure. It _might_ be an old token.